website/docs/r/environment.html.md

```diff
@@ -58,6 +58,14 @@ The `label` block supports:
 
 ~> **NOTE:** If `suspend_on_idle` and `suspend_at_time` are both null, automatic suspend is disabled. If multiple suspend or shut down options are sent in the same request, the `suspend_type` field determines which setting Skytap Cloud will honor.
 
+### Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
```

Comment:
While running the website locally the link to the wider documentation did not deep link into the correct section.  Not sure if there has been a change, but the following url/anchor took me to the operation-timeouts.

- https://www.terraform.io/docs/configuration/resources.html#operation-timeouts

Also thinking that it might be worth changing the 'actions' to 'operations'.  What do you think?

This will be the same across all resource and so I will mentioned it just here

Reply:
Changes pushed